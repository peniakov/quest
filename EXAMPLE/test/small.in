# ==========================
# Output file name header
# ==========================

ofile  = small
tdm = 0

# ==========================
# lattice dimension
# ==========================

nx     = 8
ny     = 8

# ==========================
# Hubbard model
# ==========================

U      = 2.0
t_up   = 1.0
t_dn   = 1.0
mu_up  = 0.0
mu_dn  = 0.0
L      = 20 
dtau   = 0.1  
HSF    = -1

# ==========================
# Met's algorithm
# ==========================

nwarm  = 1000
npass  = 5000
ntry   = 0
#gamma  = 0.5
#accept = 0
#reject = 0

# ==========================
# Measurements
# ==========================

nbin   = 10
tausk  = 10  
seed   = 1234567

# ==========================
# numerical
# ==========================

north  = 5 
nwrap  = 15
fixwrap = 1
errrate = 0.001
difflim = 0.00001


