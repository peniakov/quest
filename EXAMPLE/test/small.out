 2D Periodic Lattice; Nx=  8; Ny=  8; total sites=   64                          
                          U :            2.000000
                       t_up :            1.000000
                       t_dn :            1.000000
                      mu_up :            0.000000
                      mu_dn :            0.000000
             Time slice - L :           20
            Number of sites :           64
                       dtau :            0.100000
                       beta :            2.000000
     Number of warmup sweep :         1000
Number of measurement sweep :         5000
   Frequency of measurement :            1
                Random seed :      1234567
 Frequency of recomputing G :           15
Global move number of sites :            0
               Accept count :      3925244
               Reject count :      3671655
    Approximate accept rate :            0.516690
                      gamma :            0.839800
           Type of matrix B : Dense MatB  
         Type of matrix HSF : 0/1
============================================================================
 Sign of equal time measurements:
                   Avg sign :   0.10000000E+01 +-  0.00000000E+00
                Avg up sign :   0.10000000E+01 +-  0.00000000E+00
                Avg dn sign :   0.10000000E+01 +-  0.00000000E+00
============================================================================
 Equal time measurements:
          Up spin occupancy :   0.49887994E+00 +-  0.10878509E-02
        Down spin occupancy :   0.50112006E+00 +-  0.10878509E-02
               UxN_upxN_dn  :   0.38654324E+00 +-  0.84302289E-03
             Kinetic energy :  -0.14342728E+01 +-  0.25989783E-02
               Total Energy :  -0.10477296E+01 +-  0.18987718E-02
                    Density :   0.10000000E+01 +-  0.00000000E+00
                Chi_thermal :  -0.25600000E+03 +-  0.38131725E-12
              Specific heat :   0.79283404E+03 +-  0.30591963E+01
  XX Ferro structure factor :   0.25862978E+00 +-  0.27863233E-02
  ZZ Ferro structure factor :   0.24777395E+00 +-  0.11126074E-01
     XX AF structure factor :   0.11125327E+01 +-  0.19293469E-02
  Root Mean Square of XX AF :   0.11141160E+01 +-  0.19758481E-02
     ZZ AF structure factor :   0.11161612E+01 +-  0.37882473E-01
  Root Mean Square of ZZ AF :   0.12918002E+01 +-  0.59882702E-01
============================================================================
 Mean Equal time Greens function:
point ; dx=  0 ; dy=  0 :       0.50000000E+00 +-  0.00000000E+00
point ; dx=  1 ; dy=  0 :      -0.17928410E+00 +-  0.32487229E-03
point ; dx=  2 ; dy=  0 :      -0.55189618E-16 +-  0.14146790E-18
point ; dx=  3 ; dy=  0 :       0.11810448E-02 +-  0.52077458E-04
point ; dx=  4 ; dy=  0 :       0.34070431E-16 +-  0.24614704E-18
point ; dx=  1 ; dy=  1 :      -0.28557229E-16 +-  0.94210980E-19
point ; dx=  2 ; dy=  1 :       0.38733680E-01 +-  0.14703113E-03
point ; dx=  3 ; dy=  1 :       0.17039264E-16 +-  0.89711343E-19
point ; dx=  4 ; dy=  1 :      -0.40933208E-02 +-  0.34521271E-04
point ; dx=  2 ; dy=  2 :       0.16489910E-16 +-  0.16717694E-18
point ; dx=  3 ; dy=  2 :      -0.12090148E-01 +-  0.62272504E-04
point ; dx=  4 ; dy=  2 :       0.33724747E-16 +-  0.16048628E-18
point ; dx=  3 ; dy=  3 :       0.46356911E-17 +-  0.10621711E-18
point ; dx=  4 ; dy=  3 :       0.63231322E-02 +-  0.31850835E-04
point ; dx=  4 ; dy=  4 :      -0.38596516E-16 +-  0.34742899E-18
============================================================================
 Up Equal time Greens function:
point ; dx=  0 ; dy=  0 :       0.50112006E+00 +-  0.10878509E-02
point ; dx=  1 ; dy=  0 :      -0.17928410E+00 +-  0.32487229E-03
point ; dx=  2 ; dy=  0 :       0.32903585E-04 +-  0.10560348E-03
point ; dx=  3 ; dy=  0 :       0.11810448E-02 +-  0.52077458E-04
point ; dx=  4 ; dy=  0 :      -0.13078094E-03 +-  0.75971554E-04
point ; dx=  1 ; dy=  1 :      -0.54444513E-03 +-  0.40769997E-03
point ; dx=  2 ; dy=  1 :       0.38733680E-01 +-  0.14703113E-03
point ; dx=  3 ; dy=  1 :       0.97748917E-04 +-  0.78019603E-04
point ; dx=  4 ; dy=  1 :      -0.40933208E-02 +-  0.34521271E-04
point ; dx=  2 ; dy=  2 :       0.26304020E-03 +-  0.24746258E-03
point ; dx=  3 ; dy=  2 :      -0.12090148E-01 +-  0.62272504E-04
point ; dx=  4 ; dy=  2 :      -0.12692874E-03 +-  0.13542715E-03
point ; dx=  3 ; dy=  3 :      -0.68640202E-04 +-  0.78182422E-04
point ; dx=  4 ; dy=  3 :       0.63231322E-02 +-  0.31850835E-04
point ; dx=  4 ; dy=  4 :       0.18850540E-03 +-  0.20411770E-03
============================================================================
 Down Equal time Greens function:
point ; dx=  0 ; dy=  0 :       0.49887994E+00 +-  0.10878509E-02
point ; dx=  1 ; dy=  0 :      -0.17928410E+00 +-  0.32487229E-03
point ; dx=  2 ; dy=  0 :      -0.32903585E-04 +-  0.10560348E-03
point ; dx=  3 ; dy=  0 :       0.11810448E-02 +-  0.52077458E-04
point ; dx=  4 ; dy=  0 :       0.13078094E-03 +-  0.75971554E-04
point ; dx=  1 ; dy=  1 :       0.54444513E-03 +-  0.40769997E-03
point ; dx=  2 ; dy=  1 :       0.38733680E-01 +-  0.14703113E-03
point ; dx=  3 ; dy=  1 :      -0.97748917E-04 +-  0.78019603E-04
point ; dx=  4 ; dy=  1 :      -0.40933208E-02 +-  0.34521271E-04
point ; dx=  2 ; dy=  2 :      -0.26304020E-03 +-  0.24746258E-03
point ; dx=  3 ; dy=  2 :      -0.12090148E-01 +-  0.62272504E-04
point ; dx=  4 ; dy=  2 :       0.12692874E-03 +-  0.13542715E-03
point ; dx=  3 ; dy=  3 :       0.68640202E-04 +-  0.78182422E-04
point ; dx=  4 ; dy=  3 :       0.63231322E-02 +-  0.31850835E-04
point ; dx=  4 ; dy=  4 :      -0.18850540E-03 +-  0.20411770E-03
============================================================================
 Density-density correlation fn - up up:
point ; dx=  0 ; dy=  0 :       0.50000000E+00 +-  0.00000000E+00
point ; dx=  1 ; dy=  0 :       0.21392207E+00 +-  0.24289655E-03
point ; dx=  2 ; dy=  0 :       0.25042826E+00 +-  0.22932705E-03
point ; dx=  3 ; dy=  0 :       0.24984594E+00 +-  0.17376021E-03
point ; dx=  4 ; dy=  0 :       0.25045956E+00 +-  0.19310868E-03
point ; dx=  1 ; dy=  1 :       0.25142113E+00 +-  0.25303326E-03
point ; dx=  2 ; dy=  1 :       0.24769421E+00 +-  0.22052215E-03
point ; dx=  3 ; dy=  1 :       0.24995997E+00 +-  0.19150932E-03
point ; dx=  4 ; dy=  1 :       0.25013122E+00 +-  0.21501953E-03
point ; dx=  2 ; dy=  2 :       0.25030689E+00 +-  0.23926771E-03
point ; dx=  3 ; dy=  2 :       0.24952787E+00 +-  0.22180234E-03
point ; dx=  4 ; dy=  2 :       0.25029542E+00 +-  0.31341353E-03
point ; dx=  3 ; dy=  3 :       0.24995832E+00 +-  0.39688213E-03
point ; dx=  4 ; dy=  3 :       0.24968997E+00 +-  0.24333153E-03
point ; dx=  4 ; dy=  4 :       0.25004593E+00 +-  0.52723421E-03
============================================================================
 Density-density correlation fn - up dn:
point ; dx=  0 ; dy=  0 :       0.19327162E+00 +-  0.42151144E-03
point ; dx=  1 ; dy=  0 :       0.25901815E+00 +-  0.27140896E-03
point ; dx=  2 ; dy=  0 :       0.24902008E+00 +-  0.23017277E-03
point ; dx=  3 ; dy=  0 :       0.25012211E+00 +-  0.17409488E-03
point ; dx=  4 ; dy=  0 :       0.24951497E+00 +-  0.19313372E-03
point ; dx=  1 ; dy=  1 :       0.24632088E+00 +-  0.25696698E-03
point ; dx=  2 ; dy=  1 :       0.25117358E+00 +-  0.22170018E-03
point ; dx=  3 ; dy=  1 :       0.24994189E+00 +-  0.19175321E-03
point ; dx=  4 ; dy=  1 :       0.24984216E+00 +-  0.21489891E-03
point ; dx=  2 ; dy=  2 :       0.24939949E+00 +-  0.23969260E-03
point ; dx=  3 ; dy=  2 :       0.25035290E+00 +-  0.22213280E-03
point ; dx=  4 ; dy=  2 :       0.24964990E+00 +-  0.31370300E-03
point ; dx=  3 ; dy=  3 :       0.25000161E+00 +-  0.39702827E-03
point ; dx=  4 ; dy=  3 :       0.25027220E+00 +-  0.24340722E-03
point ; dx=  4 ; dy=  4 :       0.24989650E+00 +-  0.52753325E-03
============================================================================
 XX Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.61345676E+00 +-  0.84302289E-03
point ; dx=  1 ; dy=  0 :      -0.90617059E-01 +-  0.46362788E-03
point ; dx=  2 ; dy=  0 :       0.32954932E-02 +-  0.29512369E-04
point ; dx=  3 ; dy=  0 :      -0.51286705E-03 +-  0.11271252E-04
point ; dx=  4 ; dy=  0 :       0.26915466E-03 +-  0.49234016E-05
point ; dx=  1 ; dy=  1 :       0.10509996E-01 +-  0.94748187E-04
point ; dx=  2 ; dy=  1 :      -0.65711991E-02 +-  0.62264913E-04
point ; dx=  3 ; dy=  1 :       0.73351448E-03 +-  0.73601224E-05
point ; dx=  4 ; dy=  1 :      -0.28896645E-03 +-  0.40891953E-05
point ; dx=  2 ; dy=  2 :       0.18003229E-02 +-  0.24377385E-04
point ; dx=  3 ; dy=  2 :      -0.91955842E-03 +-  0.11437187E-04
point ; dx=  4 ; dy=  2 :       0.38875988E-03 +-  0.64231535E-05
point ; dx=  3 ; dy=  3 :       0.34387476E-03 +-  0.80622461E-05
point ; dx=  4 ; dy=  3 :      -0.33746330E-03 +-  0.51862732E-05
point ; dx=  4 ; dy=  4 :       0.36429157E-03 +-  0.13203091E-04
============================================================================
 ZZ Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.61345676E+00 +-  0.84302289E-03
point ; dx=  1 ; dy=  0 :      -0.90192146E-01 +-  0.10228061E-02
point ; dx=  2 ; dy=  0 :       0.28163542E-02 +-  0.91897891E-03
point ; dx=  3 ; dy=  0 :      -0.55232671E-03 +-  0.69570846E-03
point ; dx=  4 ; dy=  0 :       0.18891830E-02 +-  0.77248389E-03
point ; dx=  1 ; dy=  1 :       0.10200506E-01 +-  0.10198474E-02
point ; dx=  2 ; dy=  1 :      -0.69587379E-02 +-  0.88438999E-03
point ; dx=  3 ; dy=  1 :       0.36159014E-04 +-  0.76652420E-03
point ; dx=  4 ; dy=  1 :       0.57812123E-03 +-  0.85983652E-03
point ; dx=  2 ; dy=  2 :       0.18147926E-02 +-  0.95791121E-03
point ; dx=  3 ; dy=  2 :      -0.16500639E-02 +-  0.88786875E-03
point ; dx=  4 ; dy=  2 :       0.12910221E-02 +-  0.12542326E-02
point ; dx=  3 ; dy=  3 :      -0.86595232E-04 +-  0.15878205E-02
point ; dx=  4 ; dy=  3 :      -0.11644515E-02 +-  0.97347731E-03
point ; dx=  4 ; dy=  4 :       0.29885456E-03 +-  0.21095334E-02
============================================================================
 Average Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.61345676E+00 +-  0.84302289E-03
point ; dx=  1 ; dy=  0 :      -0.90475421E-01 +-  0.34306007E-03
point ; dx=  2 ; dy=  0 :       0.31357802E-02 +-  0.31127006E-03
point ; dx=  3 ; dy=  0 :      -0.52602027E-03 +-  0.22922047E-03
point ; dx=  4 ; dy=  0 :       0.80916411E-03 +-  0.25908657E-03
point ; dx=  1 ; dy=  1 :       0.10406832E-01 +-  0.34893932E-03
point ; dx=  2 ; dy=  1 :      -0.67003787E-02 +-  0.29335526E-03
point ; dx=  3 ; dy=  1 :       0.50106266E-03 +-  0.25737415E-03
point ; dx=  4 ; dy=  1 :       0.62778111E-07 +-  0.28716299E-03
point ; dx=  2 ; dy=  2 :       0.18051461E-02 +-  0.31751492E-03
point ; dx=  3 ; dy=  2 :      -0.11630602E-02 +-  0.29456064E-03
point ; dx=  4 ; dy=  2 :       0.68951396E-03 +-  0.41751747E-03
point ; dx=  3 ; dy=  3 :       0.20038476E-03 +-  0.53024716E-03
point ; dx=  4 ; dy=  3 :      -0.61312604E-03 +-  0.32561587E-03
point ; dx=  4 ; dy=  4 :       0.34247923E-03 +-  0.70894049E-03
============================================================================
 Onsite s-wave pair-field correlation function - site resolved:
point ; dx=  0 ; dy=  0 :       0.96635809E-01 +-  0.21075572E-03
point ; dx=  1 ; dy=  0 :       0.13529890E-01 +-  0.30781689E-04
point ; dx=  2 ; dy=  0 :      -0.27582778E-03 +-  0.16002978E-05
point ; dx=  3 ; dy=  0 :       0.15973609E-04 +-  0.42306991E-06
point ; dx=  4 ; dy=  0 :      -0.12736498E-04 +-  0.29470641E-06
point ; dx=  1 ; dy=  1 :      -0.11289945E-02 +-  0.48353933E-05
point ; dx=  2 ; dy=  1 :       0.56610221E-03 +-  0.25280040E-05
point ; dx=  3 ; dy=  1 :      -0.49072822E-04 +-  0.30926056E-06
point ; dx=  4 ; dy=  1 :       0.13312155E-04 +-  0.21075105E-06
point ; dx=  2 ; dy=  2 :      -0.14681196E-03 +-  0.10833923E-05
point ; dx=  3 ; dy=  2 :       0.59618844E-04 +-  0.44501172E-06
point ; dx=  4 ; dy=  2 :      -0.27339894E-04 +-  0.30543261E-06
point ; dx=  3 ; dy=  3 :      -0.20034695E-04 +-  0.27072655E-06
point ; dx=  4 ; dy=  3 :       0.18915545E-04 +-  0.15548580E-06
point ; dx=  4 ; dy=  4 :      -0.28781967E-04 +-  0.66137434E-06
============================================================================
 Pair-field correlation function - accumulated:
           S-Wave :             0.29822214E+00 +-  0.71023515E-03
          SX-Wave :             0.52891851E+00 +-  0.85828260E-03
           D-Wave :             0.32753324E+00 +-  0.49888359E-03
         SXX-Wave :             0.36325627E+00 +-  0.72501394E-03
         DXX-Wave :             0.36163991E+00 +-  0.82651389E-03
          PX-Wave :             0.19227033E+00 +-  0.35381422E-03
          PY-Wave :             0.19219586E+00 +-  0.32250559E-03
         PXY-Wave :             0.20672846E+00 +-  0.38596532E-03
         PYX-Wave :             0.20566511E+00 +-  0.48427779E-03
============================================================================
